src/OMSimulatorLib/System.cpp

```diff
@@ -1505,7 +1506,8 @@ oms_status_enu_t oms::System::delete_(const oms::ComRef& cref)
     for (int i=0; i<busconnectors.size()-1; ++i)
       if (busconnectors[i]->getName() == front)
       {
-        deleteAllConectionsTo(front);
+        //deleteAllConectionsTo(front);
```

Comment:
Why did you remove this call?

Reply:
i see two problems, 

1. This call actuall did not work, it did not delete the connections associated with the connector, and that is why this issue was repored
2. This call will delete all the connections assosciated with the "System or component", but that should not be the case , (e.g) we have two connection 

```
System2.C3 -> System1.C1
System2.C4 -> System1.C2
```
And now we want to delete 

`oms_delete("System2.C3")
`
The above call "deleteAllConectionsTo(front);" will delete all the connections associated with "System2" but that should not be the case , only this connection should be deleted

`System2.C3 -> System1.C1
`